### ISSUE TITLE
Crash of fitbit simulator after installation

### ISSUE THREAD

**Issue by 3v1n0** [bingtimren/fitbit-sim-starter#2]
Once installed 0.8.3 (I was prompted for upgrade) I'm seeing in the log:

```
00f2:err:winediag:SECUR32_initNTLMSP ntlm_auth was not found or is outdated. Make sure that ntlm_auth >= 3.0.25 is in your path. Usually, you can find it in the winbind package of your distribution.
```

Not sure that library should de updated in the docker...

**Comment by bingtimren** [comment]
Hi, I have updated both the script and the docker image to make it work (kind of) for the recent upgrade to 0.8.3. 

Try `npx fitbit-sim-starter -u -r`

It should work. HOWEVER, the latest fitbit simulator has something quite different from the last one, though looks simular. I have experimented with quite some options to make it work as best as I can. But still: after launching, a process consumes 100% CPU. Although your system won't lose response, you will notice the simulator a bit slow in response and your CPU cooling fan starts working. I wish someone can come up with a better docker image and I will use that.

Let me know if this works for you.

**Comment by 3v1n0** [comment]
Thanks for your work!

I didn't update the docker image yet, and staying in bionic seems something still safer for now as I couldn't run the simulator also locally in with focal + current wine.

So I'd say that it's probably better to just keep bionic + the wintricks fixes. That being said, new version works fine, but I agree that there's this CPU issue that I didn't get with bionic docker image IIRC.

**Comment by bingtimren** [comment]
Just released version 1.0.11 to provide a work around to this problem. Now you can use a '-l' option to prevent the simulator from getting update. It redirects simulator-updates.fitbit.com to 127.0.0.1.

First upgrade fitbit-sim-starter package to 1.0.11.

Then run it as follows:

`fitbit-sim-starter -i bingtimren/fitbit-simulator:linux_wine_v0_8_2 -l`

For the first time you may also want to add `-r` to reset your container.

Let me know if this solves your problem. This surely solves mine :)
